xml/article_sap_trento.xml

```diff
@@ -330,13 +330,10 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
           With the same user that installed K3s, install the &t.server; container
           using Helm:
         </para>
-        <remark>toms 2022-01-14: root or not root?</remark>
-        <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade --install \
-   <replaceable>TRENTO_SERVER_HOSTNAME</replaceable> \
+        <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade \
+   --install <replaceable>TRENTO_SERVER_HOSTNAME</replaceable> \
```

Comment:
```suggestion
   --install trento-server \
```

Reply:
Is this really correct? We had this placeholder, don't we use it?